integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSamples.java

```diff
@@ -305,6 +304,105 @@ private void createPvPvc(String domainName) {
 
   }
 
+  private void updateDomainInputsFile(String domainName, Path sampleBase) {
+    // change namespace from default to custom, set wlst or wdt, domain name, and t3PublicAddress
```

Comment:
wlst or wdt is not set here

Reply:
Thanks. I copy/pasted your original one :( will change